.drone.yml

```diff
@@ -0,0 +1,13 @@
+build:
+  test-and-lint:
+    image: node:8
+    environment:
+      - NPM_CONFIG_CACHE=/drone/.npm
+      - NPM_CONFIG_LOGLEVEL=warn
+    commands:
+      - yarn
```

Comment:
we install with `yarn` but we're caching `npm`, does this config make sense?
`NPM_CONFIG_LOGLEVEL` does it work for `yarn`?

Reply:
no I just copy-pasted it, probably to remove